conf/messages.cy

```diff
@@ -1158,6 +1158,18 @@ cease-income-sources.foreign-property.p1      = **Welsh to be translated**
 cease-income-sources.ceased-businesses.h1     = Busnesau sydd wedi dod i ben
 cease-income-sources.table-head.date-ended    = Y dyddiad dod i ben
 cease-income-sources.empty                    =
+  
+## view Income Sources
+view-income-sources.heading                  = Bwrw golwg dros eich ffynonellau incwm a’u rheoli
+view-income-sources.self-employed-h2         = Busnesau unig fasnachwr
+view-income-sources.uk-property-h2           = Eiddo yn y DU
+view-income-sources.foreign-property-h2      = Eiddo tramor
+view-income-sources.ceased-businesses-h2     = Busnesau sydd wedi dod i ben
+view-income-sources.table-head.business-name = Enw’r busnes
+view-income-sources.table-head.date-started  = Dyddiad cychwyn
+view-income-sources.table-head.date-ended    = Y dyddiad dod i ben
+view-income-sources.business-details         = Dewis
+view-income-sources.unknown-field            = Unknown
```

Comment:
for not translated into Welsh text, would be good to have similar approach as we already have like: "To be translated to Welsh" or similar, as its easy to find it

Reply:
Done